oc-manifest.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: v1
```

Comment:
Do we need both the manifest and the Helm chart? I would prefer just having one.

Reply:
The motivation here was that not everybody is using Helm, and would rather prefer using `oc`

Reply:
removed the `oc-manifest.yaml` and updated the README